admin_ui/backend/api/config.py

```
@@ -1820,3 +1823,202 @@ async def get_provider_options(provider_type: str):
+        return {
+            "status": "success",
+            "message": "Service account JSON uploaded successfully",
+            "project_id": creds.get("project_id"),
+            "client_email": creds.get("client_email"),
+        }
+        
+    except json.JSONDecodeError:
+        raise HTTPException(status_code=400, detail="Invalid JSON file")
+    except HTTPException:
+        raise
+    except Exception as e:
```

Ruff (0.15.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)